index.html

```diff
@@ -543,6 +543,111 @@ <h6>JWT Decoding</h6>
       </section>
     </section>
 
+    <section id="production">
+      <h2>Production</h2>
+
+      <p>
+        This section describes how to produce a VC-JWT encoded
+        <code>VerifiableCredential</code> from a <code>Credential</code>.
+      </p>
+
+      <p class="advisment">
+        There are currently 2 competing solutions to this problem described below. It is a goal
+        of the v2 work to resolve them and
+        reduce production rules to a single, simple, set of instructions that
+        any implementer can easily meet if they possess a software library
+        supporting [[RFC7515]] or [[RFC7519]].
+      </p>
+
+      <section>
+        <h3>In addition to...</h3>
+
+        <p>
+          There are several members (claims) of the
+          <code>Credential</code> which will need to be translated to their JOSE
+          form, and included next to the <code>vc</code> or
+          <code>vp</code> member in the <code>payload</code>.
```

Comment:
For a signed JWT, the payload of the JWS is called the "JWT Claims Set" in the JWT spec.  It would be better to use that term than "payload", I believe.

Reply:
I've made this update in https://github.com/w3c/vc-jwt/pull/11/commits/b4c079b13d7d42840c2cd40d8502cdf5103c1b6d

The only section I did not update has to do with the mapping algorithm, which is harder to read and understand when not using the word payload. If its still a blocking requirement I prefer to file an issue to discuss and refine in a seperate PR.

Reply:
let's file an issue and discuss - consistency is usually better, but not always

Reply:
https://github.com/w3c/vc-jwt/issues/36